src/make_trade.ts

```diff
@@ -29,6 +30,8 @@ export async function makeTrade(
   const [trader] = await ethers.getSigners();
   const chain = ChainUtils.fromNetwork(network);
 
+  console.log(`Using account ${trader.address}`);
```

Comment:
As a suggestion, maybe you could use the [debug](https://www.npmjs.com/package/debug) package which does nice colors and can set filters with the environment. Not needed here but something to keep in mind :smile: